components/coding/studentPortal/Layout.tsx

```diff
@@ -33,6 +37,20 @@ export const Layout: React.FC = ({ children }) => {
     fetchPolicy: "cache-and-network",
   });
 
+  const {} = useQuery<FetchUserRoleData>(FETCH_USER_ROLE, {
```

Comment:
I might move this query into the sidebar, so that the query is happening closer to the component that needs to use it.

Reply:
perfect.  did that!